composer/lib/dependabot/composer/update_checker/requirements_updater.rb

```diff
@@ -90,28 +107,33 @@ def updated_requirement(req)
               update_requirement_version(req, or_separator)
             end
 
+          # Add a T.must for new_req as it's defined in the case statement with multiple options
+          new_req = T.must(new_req)
```

Comment:
Have you [seen](https://sorbet.org/docs/exhaustiveness) `T.absurd` for case statements?

Reply:
I haven't used it before, interesting!